### PR TITLE
[WIP] getting build versions over cli

### PR DESCRIPTION
**Pull request by jiahuang**
goes with https://github.com/tessel/t2-firmware/pull/38

### CODE REVIEW

File: bin/tessel-2.js [hunk 1]
```diff
@@ -228,4 +228,35 @@ parser.command('rename')
   })
   .help("Change the name of a Tessel to something new.");
 
+parser.command('version')
+  .option('mcu', {
+    abbr: 'm',
+    flag: true,
+    help: 'The version number from the MCU (Mediatek)'
```
Comment (kevinmehall) [inline bin/tessel-2.js]: MT7620n is not a microcontroller

**Comment by kevinmehall** [comment]
The version number of the SoC OS should effectively be a version number for the openwrt-tessel repo, not the t2-firmware repo. The tessel-tools package in that OS image covers only usbexecd, spid, and tessel.js, but that version number does not cover all the other software on the system.

**Comment by johnnyman727** [comment]
@jiahuang are you still actively pursuing this feature or should I close the PR?

**Comment by jiahuang** [comment]
We do need this PR in. Path forward:
1. Add in a file during the openwrt build process. This will specify the openwrt build + the git hash of the t2-firmware repo for spid, usbexecd, and tessel js
2. the command for getting the mediatek build needs to read that file 

The code for getting the firmware from the samd doesn't need modification.

**Comment by johnnyman727** [comment]
Fixed by #231.

**Pull request closed without merging**